packages/expo-cli/src/commands/build-native/index.ts

```diff
@@ -15,17 +17,30 @@ import AndroidBuilder from './AndroidBuilder';
 import iOSBuilder from './iOSBuilder';
 import { printBuildResults, printBuildTable, printLogsUrls } from './utils';
 
-interface Options {
-  platform: 'android' | 'ios' | 'all';
+const platforms = ['android', 'ios', 'all'] as const;
```

Comment:
Maybe define an `enum` instead? 🤔 I haven't seen this syntax before.

Reply:
@dsokal I'm not sure if we want to use enums for those, especially platform type with all and without would be confusing

Reply:
I'd still use an enum. We don't need to move it to any other file (like `build-native/build.ts`). In the context of this file, it'll be pretty straightforward what's the purpose of it (and why it has the `all` value included).